flocker/acceptance/test_api.py

```diff
@@ -321,11 +324,12 @@ def query(host, port):
             req = get(
                 "http://{host}:{port}".format(host=host, port=port),
                 persistent=False
-            ).addCallback(content)
+            )
+            req.addCallbacks(content, lambda _: False)
```

Comment:
Log the reason for failure, so we can diagnose persistent failures (that will otherwise just display a timeout message).